### PR TITLE
Ruby on Rails aliases

### LINKED ISSUE #23: Add aliases for Ruby on Rails

### PR DESCRIPTION
**Pull request by lgnstn**
Ruby on Rails aliases.

Closes #23

### CODE REVIEW

**Comment by stocarul** [comment]
@al-iustin: How many of these aliases did you use so far? Do you find many of them useful?

**Comment by stocarul** [comment]
@lgnstn: Could you pleas emerge `master` into this branch?

File: bash/ruby_on_rails_aliases [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+alias rra=‘rails application’
+alias rrc=’rails console’
+alias rrd=‘rails destroy’
+alias rrn=‘rails new’
+alias rrpr=‘rails profiler’
+alias rrpl=‘rails plugin’
+alias rrs=‘rails server’
+
+# Ruby on Rails database tasks
+alias rdc=‘rake db:create’
```
Comment (stocarul) [inline bash/ruby_on_rails_aliases]: @lgnstn: Would be a problem to change these aliases in order to start with `rr` like the others in this file?

**Comment by lgnstn** [comment]
@stocarul Updated. Feel free to merge it if everything is okay.

**Review by stocarul — APPROVED** [review]
Awesome 💯

**Pull request merged**